docs/source/usage/firstread.rst

```diff
@@ -0,0 +1,322 @@
+   #first-read > .section > .section > h3 > a.headerlink {
+       display: inline-block;
+   }
+   #first-read > .section > .section > h3 > .headerlink:after {
+       visibility: hidden;
+   }
+   #first-read > .section > .section > h3:hover > .headerlink:after {
+       visibility: visible;
+   }
+   </style>
+
+Include / Import
+----------------
+
+After successfull :ref:`installation <install>`, you can start using openPMD-api as follows:
+
+C++11
+^^^^^
+
+.. code-block:: cpp
+
+   #include <openPMD/openPMD.hpp>
+
+   // example: data handling & print
+   #include <vector>   // std::vector
+   #include <iostream> // std::cout
+   #include <memory>   // std::shared_ptr
+
+   namespace api = openPMD;
+
+Python
+^^^^^^
+
+.. code-block:: python3
+
+   import openpmd_api as api
+
+   # example: data handling
+   import numpy as np
+
+
+Open
+----
+
+Open an existing openPMD series in ``data<N>.h5``.
+Further file formats than ``.h5`` (`HDF5 <https://hdfgroup.org>`_) are supported:
+``.bp`` (`ADIOS1 <https://www.olcf.ornl.gov/center-projects/adios/>`_) or ``.json`` (`JSON <https://en.wikipedia.org/wiki/JSON#Example>`_).
+
+C++11
+^^^^^
+
+.. code-block:: cpp
+
+   auto series = api::Series(
+       "data%T.h5",
+       AccessType::READ_ONLY);
+
+
+Python
+^^^^^^
+
+.. code-block:: python3
+
+   series = api.Series(
+       "data%T.h5",
+       api.Access_Type.read_only)
+
+Iteration
+---------
+
+Grouping by an arbitrary, positive integer number ``<N>`` in a series.
+Let's take the iteration ``100``:
+
+C++11
+^^^^^
+
+.. code-block:: cpp
+
+   auto i = series.iterations[100];
+
+Python
+^^^^^^
+
+.. code-block:: python3
+
+   i = series.iterations[100]
+
+Attributes
+----------
+
+openPMD defines a kernel of meta attributes and can always be extended with more.
+Let's see what we've got:
+
+C++11
+^^^^^
+
+.. code-block:: cpp
+
+   std::cout
+       << "Author: "
+       << s.author() << "\n"
+       << "openPMD version: "
+       << s.openPMD() << "\n";
+
+Python
+^^^^^^
+
+.. code-block:: python3
+
+   print(
+       "Author: {0}"
+       "openPMD version: {1}"
+       .format(
+           s.author,
+           s.openPMD))
+
+Record
+------
+
+An openPMD record can be either structured (mesh) or unstructured (particles).
+Let's read an electric field:
+
+C++11
+^^^^^
+
+.. code-block:: cpp
+
+   // record
+   auto E = i.meshes["E"];
+
+   // record components
+   auto E_x = E["x"];
+
+Python
+^^^^^^
+
+.. code-block:: python3
+
+   # record
+   E = i.meshes["E"]
+
+   # record components
+   E_x = E["x"]
+
+Units
+-----
+
+On read, units are automatically converted to SI unless otherwise specified (not yet implemented).
+Even without the name "E" we can check the `dimensionality <https://en.wikipedia.org/wiki/Dimensional_analysis>`_ of a record to understand its purpose.
+
+C++11
+^^^^^
+
+.. code-block:: cpp
+
+   // unit system agnostic dimension
+   auto E_unitDim = E.getUnitDimension();
+
+   // ...
+   // api::UnitDimension::M
```

Comment:
TODO: add pretty handlers, maybe some lookup DBs for typical quanitites, make useful.